README.md

```diff
@@ -51,3 +51,12 @@ If you make sure that PR merge permissions require:
 - a passing build (required status checks)
 
 Then this mechanism can do little to no harm, in the worst case merging a Dependabot PR you didn't intend to merge.
+
+## How to deploy
+
+- Prerequisites: a GCP project with the [cloud run and cloud build apis enabled](https://cloud.google.com/apis/docs/getting-started)
+- Create a service account in the IAM & Admin console to be used to deploy the app 
+- Create a key for the service account, this key will be configured as a secret in the GitHub actions to be able to deploy the app
+- For the service account, [grant the permissions "Service Account User", "Cloud Run Admin", "Storage Admin"](https://github.com/google-github-actions/deploy-cloudrun) and "Cloud Build Service Account", this last permission is necessary since cloud build will be used to build the image based in the source code directly
+- Configure the secrets for: GCP project id, cloud run service name, cloud run region, service account e-mail, service account key, app id and private key for the app
```

Comment:
let's use the literal names of the secrets wrapped in backticks, e.g. `GCP_PROJECT_ID`.